12-solid/3-liskov-substitution/1-example/1-example.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class VideoPlayer
```

Comment:
why is this a class and not an interface?

Reply:
We can screw up lsp in both ways, extension and implementation.
This example is about class extension, the other one is about implementation interfaces